### DISCUSSION TITLE
Version 0.21.1?

### DISCUSSION THREAD

**Discussion by bazald** [jni-rs/jni-rs#430] [Q&A] [answered]
I've noticed point releases are pretty rare the past few years. Given that 0.21.0 does not build on some architectures, any chance we'll see a 0.21.1 release before too long?

**Comment by rib** [comment]
Yeah, at the time that I spotted the regression with building on some architectures I just noticed that in the context of Android where it wasn't actually an issue (since the invocation API isn't used on Android) but certainly if this is affecting anyone I'm happy to do a release.

There was certainly a long gap where the project wasn't being actively maintained but 0.21 was released about 4 months after the 0.20 release which is hopefully more reasonable.

**Reply by bazald** [reply]
Thanks a bunch. 🥳

**Comment by rib** [comment]
For reference I've just opened this PR to make a 0.21.1 release: https://github.com/jni-rs/jni-rs/pull/431

Out of curiosity @bazald, was there a particular architecture you were building for that didn't compile?

**Reply by bazald** [reply]
You more or less guessed correctly when you observed Android is unlikely to be affected due to non-use of the invocation API. Android isn't the only OS that runs on ARM64 however.

**Comment by rib** [comment, marked as answer]
Okey I've just published a 0.21.1 release with the build fix discussed above: https://github.com/jni-rs/jni-rs/releases/tag/v0.21.1
